### DISCUSSION TITLE
tb 115.12.2 Send Later add on installed and grey - not working. Suggestions?

### DISCUSSION THREAD

**Discussion by je-github** [Extended-Thunder/send-later#720] [Q&A]
add on installed and grey - not working. Suggestions?

**Comment by jikamens** [comment]
I'm not sure what "grey" means in this context. Can you clarify? Maybe a screenshot?
Have you been able to use the add-on successfully in the past and now it's not working, or are you just starting out with it now?
How exactly did you install it?
Can you confirm what version of the add-on you have installed?

**Comment by je-github** [comment]
Gray means you click on it and nothing happens.

**Comment by jikamens** [comment]
I don't know what the "it" is that you are referring to when you say "you click on it." There are lots of things in the add-on that you could be trying to click on. 

I asked you several questions and you only answered one of them, and that one answer was vague and unhelpful.

I have neither the time nor the patience to help people who can't be troubled to put in the minimal effort to help me help them.

Answer my questions, post some screenshots or something to make it clear what you're asking, and I will do my best to help you.

On July 17, 2024 8:31:15 AM EDT, je-github ***@***.***> wrote:
>Gray means you click on it and nothing happens.
>
>On Wed, Jul 17, 2024, 8:18 AM Jonathan Kamens ***@***.***>
>wrote:
>
>> I'm not sure what "grey" means in this context. Can you clarify? Maybe a
>> screenshot?
>> Have you been able to use the add-on successfully in the past and now it's
>> not working, or are you just starting out with it now?
>> How exactly did you install it?
>> Can you confirm what version of the add-on you have installed?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Extended-Thunder/send-later/discussions/720#discussioncomment-10073040>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AUC62FHW6UZNZENXNEGBOLDZMZOI5AVCNFSM6AAAAABK75DPRSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBXGMYDIMA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***
>> .com>
>>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/Extended-Thunder/send-later/discussions/720#discussioncomment-10073184
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
-- 
Sent from my phone. Please excuse brevity and autocorrect errors.
